### ISSUE TITLE
Support directed graphs

### ISSUE THREAD

**Issue by iankloo** [iankloo/sigmaNet#12]
Support directed graphs with edge arrows. 

Based off conversation here: #10

**Comment by aj2duncan** [comment]
I think adding support for directed graphs would be great. I wonder also if you might consider adding the ability to choose *in* or *out* degree as well as *all* for node sizes. 

Something like adding an argument to addNodeSize() which defaults to *all* but allows users to choose in or out. 

Apologies if you've already thought of this and I'd be happy to help once you're happy with the directed graphs.

**Comment by iankloo** [comment]
Hey - thanks again for looking into this issue.  I found that the real problem with the node order after finding betweenness, as you found, is the way that directed vs. undirected graphs are handled.  When I went to work on the directed graph functionality, I found that the problem was fixed if you specify whether or not your graph is directed any time an igraph is created or taken apart.  This seems to have fixed the previous issue you identified as well as add support for directed graphs.

I have a little more testing to do on the arrow implementation before I merge the dev branch, but this enhancement is all but done.
